backend/src/test/java/gov/cdc/usds/simplereport/api/featureflags/FeatureFlagsControllerTest.java

```diff
@@ -0,0 +1,27 @@
+package gov.cdc.usds.simplereport.api.featureflags;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import gov.cdc.usds.simplereport.config.FeatureFlagsConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+class FeatureFlagsControllerTest {
+
+  private FeatureFlagsController featureFlagsController;
+  private FeatureFlagsConfig _mockFeatureFlagConfig = new FeatureFlagsConfig();
+
+  @BeforeEach
+  public void setup() {
+    _mockFeatureFlagConfig.setMultiplexEnabled(true);
+    this.featureFlagsController = new FeatureFlagsController();
+    ReflectionTestUtils.setField(
+        this.featureFlagsController, "featureFlags", _mockFeatureFlagConfig);
+  }
+
+  @Test
+  public void endpointReturnsFeatureFlagsConfigObj() {
```

Comment:
Can you remove the `public` modifier on this test?

Reply:
Removing the public modifier makes random backend test start failing so I am not sure how how to move forward without it

Reply:
added comments